### DISCUSSION TITLE
Nextcloud AIO "502 Bad Gateway" after crash

### DISCUSSION THREAD

**Discussion by chrisborell** [nextcloud/all-in-one#3508] [Questions] [answered]
Hello everyone,

I'm facing an issue with my Nextcloud All-In-One (AIO) installation, which has crashed overnight for the second time and won't come back online. I've checked the logs, but there doesn't seem to be any clear indication of what's causing the problem. The AIO instance itself indicates that everything is running and ready to launch. I'm currently getting a "502 Bad Gateway, openresty."

I'm running Nexcloud AIO on Unraid, Nginx Proxy Manager and all was working smoothly until something happened overnight! 

Could someone please take a look at the attached logs and help me identify what might be going wrong? Your assistance would be greatly appreciated.  

<img width="652" alt="Screenshot 2023-10-09 at 4 45 32 pm" src="https://github.com/nextcloud/all-in-one/assets/52059185/7a5149ae-dd5d-4064-960e-1f1a8c3e93a2">
  
[nextcloud-aio-mastercontainer.txt](https://github.com/nextcloud/all-in-one/files/12842935/nextcloud-aio-mastercontainer.txt)
[nextcloud-aio-apache.txt](https://github.com/nextcloud/all-in-one/files/12842936/nextcloud-aio-apache.txt)
[nextcloud-aio-nextcloud.txt](https://github.com/nextcloud/all-in-one/files/12842937/nextcloud-aio-nextcloud.txt)

**Comment by szaimen** [comment, marked as answer]
Hi, see https://github.com/nextcloud/all-in-one/issues/3491

**Reply by szaimen** [reply]
This is now fixed with v7.4.1 Beta. Testing and feedback is welcome! See https://github.com/nextcloud/all-in-one#how-to-switch-the-channel
